### ISSUE TITLE
revise documentation for microDL (dependencies: tensorflow 1.13, keras 2.1.6)

### ISSUE THREAD

**Issue by mattersoflight** [mehta-lab/microDL#161]
The current microDL codebase needs a refreshed documentation to ease the on-boarding of new users to the current pipeline and to guide the transition of the code to tensorflow 2.0 or pytorch(#141) .

The structure of documentation we are aiming for is as follows:  Simplify readme to describe installation and how to use CLI for preprocessing, training, inference. The readme also provides an index of other documentation. Most of the documentation will be centralized in the docs folder, containing 

- [x] the [notebook](https://github.com/czbiohub/microDL/tree/master/notebooks) used for image translation exercise at the deep learning course. The exercise should be updated in following ways:
     - a graphical abstract near the top.
     - note checkpoints that clarify when and how the user should examine the data and model before proceeding. 
     - (nice to have) illustrate how to write a custom training loop.
- [x] annotated list of configs used for specific projects: 
     - virtual staining as reported in our [paper](https://elifesciences.org/articles/55502) with 2D, 2.5D, and 3D U-Nets. 
     - refinements made for robust virtual staining to train models with data acquired during March/April 2022.
     - models that turn out to be valuable.
- [x] summary of algorithms and approaches implemented in the code base. A good example is description of [learning rate scheduler](https://github.com/czbiohub/microDL/blob/master/micro_dl/train/readme.md). 
     
Since the docs folder is versioned with the repo, this approach will allow us to version the documentation along with the code.

**Comment by mattersoflight** [comment]
PR #162 addresses this issue.
